version.json

```diff
@@ -1,3 +1,3 @@
 {
-  "version": "v1.32.0-rc3"
```

Comment:
Non-blocker/A small footnote here: I think the way releaser workflow works is that it doesn't include the PR that bumps the version number in the release notes of the release being made. This is because, it creates a draft release _before_ the version bump is merged.

This behaviour is fine when the PR only bumps the version. But _this_ PR does more: it also upgrades dependencies and as a consequence bumps go version.

If you'd like this change to appear in the release note: either separate the PRs, or manually update the generated release note to state this. 

I personally, opt for separate PRs.

Reply:
Ah, that is correct. I will separate the PRs